region/brinstar/green.json

```diff
@@ -3549,6 +3549,19 @@
                   "name": "Gate Glitch",
                   "notable": false,
                   "requires": ["h_canBlueGateGlitch"]
+                },
+                {
+                  "name": "Green Hills Grapple Gate Glitch",
+                  "notable": true,
+                  "requires": [
+                    "Grapple",
+                    "SpeedBooster",
+                    "canBePatient"
```

Comment:
With `canBePatient`, this will be at least in `Extreme`. Do you think that's accurate?
```suggestion
                    "canTrickyJump",
                    "canBePatient"
```

Reply:
Yeah, It will need a setup of some kind to not try over and over.  And then we can take off canBePatient